code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -502,7 +502,7 @@
 		if(HAS_TRAIT(src, TRAIT_OVERSIZED) && !HAS_TRAIT(helper, TRAIT_OVERSIZED))
 			visible_message(span_warning("[helper] tries to pat [src] on the head, but can't reach!"))
 			return
-		else if(HAS_TRAIT(src, TRAIT_QUICKREFLEXES) && (src.stat != UNCONSCIOUS) && !src.incapacitated(IGNORE_RESTRAINTS))
+		else if(HAS_TRAIT(src, TRAIT_QUICKREFLEXES) && (src.stat != UNCONSCIOUS) && !src.incapacitated(!IGNORE_RESTRAINTS))
```

Comment:
You can't use regular logic operands on bitflags, this just sends a zero

Reply:
I'm stupid, and yeah this was the wrong thing